### ISSUE TITLE
Parsing (WCAG 4.1.1)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1244]
**Request:** The intent of this criterion is to ensure that user agents, including assistive technologies, can accurately interpret and parse content. If the content cannot be parsed into a data structure, then different user agents may present it differently or be completely unable to parse it. Some user agents use "repair techniques" to render poorly coded content. 

**Issue:**
Validating the pages with the W3C Validator results in errors. If possible, the errors should be corrected.

Validator: https://validator.w3.org/nu/

Pages:
- https://slidewiki.org/ 
- https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3 
- https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit 
- https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/markdownEdit 
- https://slidewiki.org/search?keywords=accessibility&sort=score 
- https://slidewiki.org/user/vivien.zschammer/recommendations 
- https://slidewiki.org/addDeck 
- https://slidewiki.org/contactus 
- https://slidewiki.org/deck/127411/test/deck/127411 
- https://slidewiki.org/accessibility 
- https://slidewiki.org/user/vivien.zschammer/settings/profile
- https://slidewiki.org/user/vivien.zschammer/stats

**Comment by VivienDZB** [comment]
There is a new way to test this:
Check the page with the validator: https://validator.w3.org/nu/
Run the bookmarklet "Check for WCAG 2.0 parcing compliance", which can be found here: https://validator.w3.org/nu/about.html#extras

The following pages still have syntax issues:
Add Deck - https://slidewiki.org/addDeck (number = 3)
Editor - https://slidewiki.org/deck/131649-1/_/slide/856768-2/856768-2:2/edit (number = 9)
Markdown editor - https://slidewiki.org/deck/131649-1/_/slide/856768-2/856768-2:2/markdownEdit (number = 9)
Presentation view - https://slidewiki.org/presentation/131649/_/131649/856767-2/#/slide-856767-2 (number = 16)
Contact us - https://slidewiki.org/contactus (number = 1)

**Comment by aoelen** [comment]
Fixed in #1283
